.travis.yml

```diff
@@ -1,10 +1,6 @@
 language: node_js
 node_js:
-- node
-test:
-- yarn check
-- yarn checkPeerDependencies
```

Comment:
checkPeerDependencies does not make sense anymore.
yarn check is broken at the moment